src/mixins/amalgamation-mixin.ts

```diff
@@ -27,90 +27,100 @@ export default class AmalgamationMixin extends Vue {
   ]
 
   // *** I'M STILL WONDERING IF I WANT TO USE THESE
-  // readonly isLear = (item: AmalgamatingBusinessIF): boolean => (item?.type === 'lear')
-  // readonly isForeign = (item: AmalgamatingBusinessIF): boolean => (item?.type === 'foreign')
```

Comment:
I can't use these after all. They look slick but if I use them in an IF statement before dereferencing a lear or foreign property then VS Code doesn't trust that I know what I'm doing and reports a type error.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/fb303f91-e742-42fb-9c4d-f5c0fa2f0b0d)

Reply:
Experiencing the same thing. I casted the business as any when trying to do something similar.

Reply:
Well I **am** using the current best practice for union types (ie, having a type variable). I guess VS Code trusts us in the same statement but not if we get more abstract.